azure-pipelines-release.yml

```diff
@@ -0,0 +1,730 @@
+
+    - task: DotNetCoreCLI@2
+      displayName: 'Publish Reservations Api'
+      inputs:
+        command: publish
+        publishWebProjects: false
+        projects: 'src/SFA.DAS.Reservations.Api/SFA.DAS.Reservations.Api.csproj'
+        arguments: '--configuration $(buildConfiguration) --output $(build.artifactstagingdirectory)/publish --no-restore --no-build'
+
+    - task: DotNetCoreCLI@2
+      displayName: 'Publish Employer Incentives Api'
+      inputs:
+        command: publish
+        publishWebProjects: false
+        projects: 'src/SFA.DAS.EmployerIncentives.Api/SFA.DAS.EmployerIncentives.Api.csproj'
+        arguments: '--configuration $(buildConfiguration) --output $(build.artifactstagingdirectory)/publish --no-restore --no-build'
+      
+    - task: DotNetCoreCLI@2
+      displayName: 'Publish Find Epao Api'
+      inputs:
+        command: publish
+        publishWebProjects: false
+        projects: 'src/SFA.DAS.FindEpao.Api/SFA.DAS.FindEpao.Api.csproj'
+        arguments: '--configuration $(buildConfiguration) --output $(build.artifactstagingdirectory)/publish --no-restore --no-build'
+
+    - task: CopyFiles@2
+      displayName: 'Copy Files to: $(build.artifactstagingdirectory)'
+      inputs:      
+        Contents: |
+          azure/**          
+        TargetFolder: '$(build.artifactstagingdirectory)/publish'
+        OverWrite: true
+
+    - task: PublishPipelineArtifact@1
+      displayName: 'Publish Artifact'
+      inputs:
+        targetPath: '$(build.artifactstagingdirectory)/publish'
+        artifactName: 'ApimEndpointsArtifacts'
+
+- stage: Deploy_AT
+  dependsOn: Build
+  displayName: Deploy to AT
+  pool:
+    name: DAS - Continuous Deployment
+  variables:
+  - group: DEV das-apim-endpoints
+  - group: DevTest Management Resources
+  - group: AT DevTest Shared Resources
+  jobs:
+  - template: azure-pipelines-templates/deploy/job/arm-deploy.yml@das-platform-building-blocks
+    parameters:
+      ServiceConnection: SFA-DAS-DevTest-ARM
+      SubscriptionId: $(SubscriptionId)
+      Location: $(ResourceGroupLocation)
+      Environment: DEV
+      TemplatePath: $(Pipeline.Workspace)/ApimEndpointsArtifacts/azure/template.json
+      ParametersPath: $(Pipeline.Workspace)/ApimEndpointsArtifacts/azure/template.parameters.json
+      ConfigSchemaPath: $(Pipeline.Workspace)/das-employer-config/Configuration/das-apim-endpoints
+      TemplateSecrets:
+        LoggingRedisConnectionString: $(LoggingRedisConnectionString)
+        ConfigurationStorageConnectionString: $(ConfigurationStorageConnectionString)
+
+  - deployment: Deploy_FindApprenticeshipTraining
+    dependsOn: DeployInfrastructure
+    condition: and(succeeded(), eq(${{ parameters.FindApprenticeshipTraining }}, true))
+    variables:
+      ResourceGroupName: $[dependencies.DeployInfrastructure.outputs['DeployInfrastructure.ArmOutputs.ResourceGroupName']]
+    environment: DEV
```

Comment:
Should the environment be called DEVTEST (or maybe DevTest) to be consistent with the way these environments (AT, TEST, PP) are generally referred to?

Reply:
DevTest should refer to our subscription that deploys to AT TEST TEST2 and DEMO but yeah we do generally include PP when we refer to DevTest (to make matters more confusing!)

Going forward we could correct this and have a DEV environment to deploy to AT TEST TEST2 PP.
PROD -> Production stages
MO -> MO stages

Reply:
Sounds reasonable